package.json

```diff
@@ -70,7 +70,7 @@
   "scripts": {
     "clean": "rimraf out/server && rimraf lib",
     "compile": "tsc -p .",
-    "lint": "npm ci && pre-commit run -a",
+    "lint": "npm ci && ./scripts/build-syntaxes && pre-commit run -a",
```

Comment:
Why is this necessary before pre-commit?

Reply:
pre-commit will fail it if files are touched, including a nice diff. In the future I would rewrite it as pure python and make it a local hook but I do not have the energy to do it. It seems every minute I look over that repo I find other things that have zero CI testing.

Reply:
Got it, thanks!